recipe/common.php

```diff
@@ -118,6 +118,7 @@
 option('tag', null, InputOption::VALUE_OPTIONAL, 'Tag to deploy');
 option('revision', null, InputOption::VALUE_OPTIONAL, 'Revision to deploy');
 option('branch', null, InputOption::VALUE_OPTIONAL, 'Branch to deploy');
+option('hooks', null, InputOption::VALUE_OPTIONAL, 'Set to false to skip after/before hooks', true);
```

Comment:
Need to change to `no-hooks`. Also please set $mode = InputOption::VALUE_NONE and remove default value. Then in TaskCommand:66 you can use just one simple condition `if ($input->getOption('no-hooks')) {`.